src/bindings/cpp/tests/testcpp_key.cpp

```diff
@@ -351,6 +351,21 @@ TEST (key, exceptions)
 	{
 		succeed_if (test.getName () == "user:/name", "name not unchanged");
 	}
+
+	try
+	{
+		test.addName ("no");
+	}
+	catch (kdb::KeyInvalidName const &)
+	{
+		succeed_if (test.getName () == "user:/name", "name not unchanged");
```

Comment:
The error message is actually quite unhelpful:

```suggestion
		succeed_if (test.getName () == "user:/name", "name not unchanged after trying to add name to invalid named key");
```

Please also correct other unhelpful messages above.